### DISCUSSION TITLE
Possible way to protect solo

### DISCUSSION THREAD

**Discussion by kasibarndoor** [GerryDazoo/Slinger#393] [Ideas]
I purchased a second hand solo and connected it up to a router and looked with Wireshark what was going on, it issued two ARP requests one was "who has got IP address 192.168.0.55 tell 192.168.0.101" the other was "whose got 192.168.0.1 tell 192.168.0.101".  I assume it has been configured with a fixed IP address and a fixed Gateway address but was given 101 by the DHCP server on the router.

The first was easily fixed by allocating 192.168.0.55 to the solos Mac address, the thought I have is what if I configure the router to have a different gateway address and to allocate the slinger server Mac address to 192.168.0.1?  This would mean the server would have to handle the phone home messages as well as media packets

I do not know whether the Solo is bricked yet but it might be worth trying with a known unbricked one. Perhaps with two routers one could create a DMZ and look at the replies from Mr Sling!

**Comment by sp222222** [comment]
setup the solos with internet unplugged. make blocking rules with whatever static IP you assign each slingbox. don’t turn internet back on until you confirm solos are static and match the blocking rules you made. Trust me, I lost three solos because I only had inbound blocked. Port 5678 must be blocked at all costs. It bricks the solos, av and tuner boxes.
